sender.go

```diff
@@ -52,9 +62,34 @@ var gcmSendEndpoint = GcmSendEndpoint
 //	}
 type Sender struct {
 	ApiKey string
+	URL    string
 	Http   *http.Client
 }
 
+// NewClient returns a new sender with the given URL and apiKey.
+// If one of input is empty or URL is malformed, returns error.
+// It sets http.DefaultHTTP client for http connection to server.
+// If you need our own configuration overwrite it.
+func NewClient(urlString, apiKey string) (*Sender, error) {
```

Comment:
For user, `NewClient` is much more sense for me (e.g., you use GCM**Client** https://github.com/mercari/gaurun/blob/master/gaurun/client.go#L36 If I were you I would name same way)